docs/vsphere/main.md

```diff
@@ -0,0 +1,40 @@
+# vSphere provider
+
+## Prerequisites
+
+1. `kubectl` CLI installed locally.
+2. vSphere instance version `6.7.0` or higher.
+3. vSphere account with aproriate priveleges.
```

Comment:
@a13x5 can we attach short list of what privileges are needed? For aws/azure we basically added links to the official docs on what resources / configs have to be enabled, we need something enabling user like this here.

note: plz check the spelling for `privileges`, you seem to have typo

Reply:
@DinaBelova I couldn't find exact set of permissions in CAPV docs. Permissions are documented correctly only for CSI driver.
I had an Idea to provide short list of privileges based on my role, but then I remembered that there is no such thing as "short list of privileges" in vsphere. I will try to add general overview of the permissions needed, without concrete stuff